recipes/example/meta.yaml

```diff
@@ -9,7 +11,7 @@ source:
     fn: simplejson-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
```

Comment:
My leaning is keep the sample a little simpler, but I'm hoping to add another example today too.